Xlayton/Helium
# Basic Room Functionality and Public Servers Discoverability.

Added in this update:
Room Creation
Joining Rooms
Limiting what rooms you can join to what rooms you are in

To Test:
Drop/Create Tables using new schema
Make an account
Log into said account
Make a room
Copy invite code
In a different browser make a new account, log in and then paste the url in your browser
It should redirect to the homepage with the room now on the left-hand side.
Boom you have rooms.
